lib/oli_web/templates/layout/_delivery_footer.html.eex

```diff
@@ -17,6 +18,7 @@
     <div class="text-right">
     </div>
   </div>
+    <%= render_license(assigns) %>
```

Comment:
Instead of passing all the assigns to this function component, we could do `<%= render_license(assigns[:license]) %>` and define the fc as follows:
```elixir
def render_license(%{license: %{license_type: cc_license}} = assigns) do
   ...
end

def render_license(nil), do: nil
```